collector/env.py

```diff
@@ -11,3 +11,6 @@
 INPUT_DATA_FORMAT = os.environ.get('INPUT_DATA_FORMAT', '').upper()
 APP_NAME = os.environ.get('APP_NAME')
 SSL_VERIFY = os.environ.get('SSL_VERIFY', 'true').lower() in ['true', 'y']
+REDIS_ENV = os.environ.get('REDIS_ENV', '{"host": "redis"}')
+REDIS_PASSWORD = os.environ.get('REDIS_PASSWORD', 'smartvm')
```

Comment:
While this is not supposed to be a super secret kind of password, I still think it would be a good idea to keep the code devoid of any hard-coded passwords.

If it's going to be populated via e2e-deploy then why set it here?

Reply:
That's a good point, but I think we should keep that for a future PR (or maybe not even at all), the Redis is accessible within the cluster only anyways. And we kinda need to have it available there now... It would require defining a secret etc. and since this would go away with the rearch, I think we can live with it as it is for now...

Reply:
Ok, sounds good, this works for now!